lib/locale/pl/strings.php

```diff
@@ -35,6 +35,7 @@
 $_LANG['Other'] = 'Inny';
 $_LANG['Offer'] = 'Oferta';
 $_LANG['Fault'] = 'Usterka';
+$_LANG['Ticket deadline can be changed only by verifier if verifier is set'] = 'Termin ostateczny może zmienić jedynie weryfikator jeśli weryfikator jest ustawiony';
```

Comment:
Lepiej tak:
If verifier is set then he's the only person who can change deadline.
Jeśli weryfikator jest ustawiony, jest on jedyną osobą, która może zmienić termin ostateczny.